src/datasource.ts

```diff
@@ -583,6 +735,9 @@ export async function getDataQueryRequestItems(
       items = await formQueriesForExpression(expr, target, connector, defaultInterval);
       break;
     }
+    case Tab.Relationships: {
+      break;
```

Comment:
this code isn't supposed to be here. `getDataQueryRequestItems` only works with timeseries targets